cmd/config/docs/api-conventions/functions-spec.md

```diff
@@ -1,13 +1,18 @@
-# Configuration Functions Specification
+# KRM Functions Specification
+
+_apiVersion: v1_
+
+## Overview
 
 This document specifies a standard for client-side functions that operate on
-Kubernetes declarative configurations. This standard enables creating
-small, interoperable, and language-independent executable programs packaged as
-containers that can be chained together as part of a configuration management pipeline.
-The end result of such a pipeline are fully rendered configurations that can then be
-applied to a control plane (e.g. Using ‘kubectl apply’ for Kubernetes control plane).
-As such, although this document references Kubernetes Resource Model and API conventions,
-it is completely decoupled from Kubernetes API machinery and does not depend on any
+Kubernetes declarative configurations referred to as _KRM Functions_. This
```

Comment:
Are the declarative configurations called "KRM Functions" or does that refer to the programs? I'd argue it's the latter, as this sentence originally implied.